src/main/java/com/internhub/data/scrapers/ScraperUtils.java

```diff
@@ -0,0 +1,79 @@
+package com.internhub.data.scrapers;
+
+import org.jsoup.select.Elements;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class ScraperUtils {
+
+    public static final Logger logger = LoggerFactory.getLogger(ScraperUtils.class);
+
+    public static boolean isInCompanyDomain(URL url, URL companyURL, String companyAbbrev) {
+        return url.getHost().equals(companyURL.getHost()) || url.getHost().contains(companyAbbrev);
+    }
+
+    public static Elements fixHTMLBody(Elements html) {
+        return removeFromHTMLBody(html, "header", "navigation", "footer", ".footer");
+    }
+
+    public static Elements removeFromHTMLBody(Elements html, String... queries) {
+        for (String query : queries) {
+            Elements scrap = html.select(query);
+            if (scrap != null)
+                scrap.remove();
+        }
+        return html;
+    }
+
+    public static String getBase(String link) {
+        URL url = makeURL(link);
+        if (url == null)
+            return null;
+        return url.getProtocol() + "://" + url.getHost();
+    }
+
+    public static URL makeURL(String link) {
+        try {
+            return new URL(link);
+        } catch (MalformedURLException ex) {
+            // logger.error(link + " is malformed.", ex);
```

Comment:
Don't just comment out code, if you are going to remove it then remove it. This message is useful to have, but perhaps it would be better to have as a log statement and without the extra exception information.